app/mikane/src/app/pages/account/change-password/change-password.component.ts

```diff
@@ -36,8 +36,8 @@ export class ChangePasswordComponent implements OnInit, OnDestroy {
 
 	changePasswordForm = new FormGroup({
 		currentPassword: new FormControl<string>('', [Validators.required]),
-		newPassword: new FormControl<string>('', [Validators.required]),
-		newPasswordRetype: new FormControl<string>('', [Validators.required]),
+		newPassword: new FormControl<string>('', [Validators.required, Validators.minLength(3)]),
+		newPasswordRetype: new FormControl<string>('', [Validators.required, Validators.minLength(3)]),
```

Comment:
Should probably have error messages in templte. Looks like there's only one for `invalid` now, but `required`, `minLength` and `match_error` exist

Reply:
Tried using `minLength`, but couldn't get it to work for some reason. Can try again.

Reply:
Might be `minlength` (small L)